src/content/intro/introText.coffee

```diff
@@ -184,7 +184,7 @@ Job.IntroTakeOver::next = Page.IntroTakeOver = class IntroTakeOver extends Page
     --> `L A bra, mistress.`
 
     --> `D Right. A bra.`
-    -- `D You, slave, she's not to be without a cock inside her for twenty four hours.` I order one of my male slaves to attend to her. `D After that you, bra-girl, return to the front desk. This place is closed for repairs until further notice, no more customers, but keep everything running like normal.`
+    -- `D You, slave, she's not to be without a cock inside her for twenty four hours.` I order one of my male slaves to attend to her. `D After that - you, bra-girl, return to the front desk. This place is closed for repairs until further notice. No more customers, but keep everything running like normal.`
```

Comment:
The dash here seems rather odd. Should be a comma instead. I'm still going to merge the PR, just commenting here on things I'll be changing in my own commit shortly.